azure-pipelines.yml

```diff
@@ -3,7 +3,7 @@ variables:
   - name: products
     value: 6eff390d-80c0-4456-81b6-6abafa71e768
   - name: tags
```

Comment:
I guess you can just remove the tags session. Without it, by default it is not production